### PR TITLE
a lot of thread housekeeping

### PR DESCRIPTION
**Pull request by caveman99**
Switch them off when not needed / disabled.

### CODE REVIEW

**Comment by thebentern** [comment]
@caveman99 I replaced those instances with disable and I also made it return an int32_t so that we could return it in a runOnce

**Comment by caveman99** [comment]
Every time we call this, same leak, 360 bytes...
`[NodeInfoModule] Thread NodeInfoModule leaked heap 90348 -> 89988 (-360)`

Also this, but only sometimes, still same amount of bytes when it happens
`[RadioIf] Thread RadioIf leaked heap 88224 -> 87900 (-324)`

File: src/modules/SerialModule.cpp [hunk 1]
```diff
@@ -221,8 +221,7 @@ int32_t SerialModule::runOnce()
         return (10);
     } else {
         DEBUG_MSG("Serial Module Disabled\n");
-
-        return INT32_MAX;
+        return disable();
```
Comment (caveman99) [inline src/modules/SerialModule.cpp:224]: I think these "We dont want that" noisy bits can go ...

File: src/modules/esp32/AudioModule.cpp [hunk 2]
```diff
@@ -259,7 +260,7 @@ int32_t AudioModule::runOnce()
         return 100;
     } else {
         DEBUG_MSG("Audio Module Disabled\n");
-        return INT32_MAX;
+        return disable();
```
Comment (caveman99) [inline src/modules/esp32/AudioModule.cpp:263]: Same here, this was good in development, but not any more

**Comment by caveman99** [comment]
> Also this, but only sometimes, still same amount of bytes when it happens `[RadioIf] Thread RadioIf leaked heap 88224 -> 87900 (-324)`

the 324 may be harmless... Router frees that up again, seems to be a packet struct...

```
??:??:?? 16 [RadioIf] (bw=250, sf=11, cr=4/8) packet symLen=8 ms, payloadSize=27, time 690 ms
??:??:?? 16 [RadioIf] Lora RX (id=0x7cf711c6 fr=0x24 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=10.5 rxRSSI=1.84221)
??:??:?? 16 [RadioIf] AirTime - Packet received : 690ms
??:??:?? 16 [RadioIf] --- --- Thread RadioIf leaked heap 120040 -> 119716 (-324)
??:??:?? 16 [Router] Rx someone rebroadcasting for us (id=0x7cf711c6 fr=0x24 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=10.5 rxRSSI=1.84221)
??:??:?? 16 [Router] didn't find pending packet
??:??:?? 16 [Router] Found existing packet record for fr=0x25d63124,to=0xffffffff,id=0x7cf711c6
??:??:?? 16 [Router] Found existing packet record for fr=0x25d63124,to=0xffffffff,id=0x7cf711c6
??:??:?? 16 [Router] Add packet record (id=0x7cf711c6 fr=0x24 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=10.5 rxRSSI=1.84221)
??:??:?? 16 [Router] Ignoring incoming msg, because we've already seen it (id=0x7cf711c6 fr=0x24 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=10.5 rxRSSI=1.84221)
??:??:?? 16 [Router] Incoming message was filtered 0x25d63124
??:??:?? 16 [Router] +++ +++ Thread Router freed heap 119716 -> 120040 (324)
```

**Pull request merged**